lexers/g/go.go

```diff
@@ -13,7 +13,7 @@ var Go = internal.Register(MustNewLexer(
 	&Config{
 		Name:      "Go",
 		Aliases:   []string{"go", "golang"},
-		Filenames: []string{"*.go"},
+		Filenames: []string{"*.go", "go.mod"},
```

Comment:
Didn't find it on pygments nor Chroma.

Reply:
This is wakatime specific :wink:

Reply:
https://github.com/wakatime/wakatime/blob/2e636d389bf5da4e998e05d5285a96ce2c181e3d/wakatime/stats.py#L173

Reply:
I'm not too sure we have to change Chroma's implementation adding WakaTime specific needs to it. If it's really necessary and it's missing on Pygments let's open a PR to them and discuss this specific one before merging it.

Reply:
After your comment, I thought about it. If we consider, that pygments and chroma are used for language detection as well as language parsing and highlighting, it becomes obvious, that `go.mod` does not fit into golang lexer, as it does not contain golang code. Will move it into wakatime-cli code. :+1: